### DISCUSSION TITLE
Multi stimuli and multi response of different type

### DISCUSSION THREAD

**Discussion by SearaChen** [jspsych/jsPsych#596] [Q&A]
Is there anyway I can implement multi stimuli and multi response, while two stimuli are of different two types (one audio, one text),  and two response are of different types (one slider and one  button) ? 

Use-Case: i want the trial to play an audio stimuli while having the corresponding text, and have the subject rate on a sliding scale, and have a button for replaying audio stimuli

- There doesn't seem to be any documentation regarding this issue? 
- Why is it that the multi-stim-multi-response plug-in doesn't seem to come with the jspsych-6 package? (I have only seen it mentioned in JsPsych-4 versions)
- So far I have only been working with a workaround that is having all responses to be in buttons,  and I cannot insert any text stimuli : https://github.com/SearaChen/Psych_Experiment_Prototype/blob/master/audio_experiment.html

Thank you!

**Comment by jodeleeuw** [comment]
This sounds like you will need a custom plugin. You could start with the audio-slider-response plugin and use the `prompt` parameter for displaying text. The only thing you need to add is an extra button that will play the audio again. You can look at how the button is configured to continue the trial, and perhaps modify that to play the audio file again.

**Comment by jodeleeuw** [comment]
The `multi-stim-multi-response` plugin was removed because it was not widely used and supporting it in newer version of jsPsych became challenging. Most of the use cases for that plugin were achievable by combining other plugins together in a timeline.

**Comment by SearaChen** [comment]
Okay that is very clear, thank you josh!
